util/util.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+func Arrayfilter(ss [] interface {}, test func(interface{}) bool) (ret []interface{}) {
```

Comment:
What Arrayfilter do? what do ss and test represent?

The idea of using short variable names is to avoid asking these kinds of questions. When the code/logic is not self-explanatory itself these questions might arise for other people.

Reply:
I was searching for how to review struct properties dynamically, I ended up using auxiliary package for this, so I'm removing this code.